src/main/resources/add-ons-to-index.json

```diff
@@ -614,6 +635,9 @@
       "uid": "org.openmrs.module.databasemessages",
       "type": "OMOD",
       "name": "Database Messages",
+	  "tags": [
+        "database"
```

Comment:
"database" is not a helpful tag. It's not a category of addons that's useful to group together.